src/components/Tooltip/Tooltip-test.js

```diff
@@ -61,24 +61,33 @@ describe('Tooltip', () => {
 
   describe('events', () => {
     it('changes state on click', () => {
-      const wrapper = mount(<Tooltip triggerText="Tooltip" />);
+      const wrapper = mount(<Tooltip showIcon={false} triggerText="Tooltip" />);
       const trigger = wrapper.find('.bx--tooltip__trigger');
-      trigger.simulate('click');
-      expect(wrapper.state().open).toEqual(false);
```

Comment:
some of these tests didn't look like they were working right before, i fixed up the logic so they're actually testing what they say they're testing